CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 Changes by Version
 ==================
 
-1.23.0 (2021-07-11)
+1.24.0 (2021-07-08)
+-------------------
+* Include OIDC plugin in binary ([#1501](https://github.com/jaegertracing/jaeger-operator/pull/1501), [@esnible](https://github.com/esnible))
+* Update jaeger operator to support strimzi operator 0.23.0 ([#1495](https://github.com/jaegertracing/jaeger-operator/pull/1495), [@rubenvp8510](https://github.com/rubenvp8510))
+* Feature/add deployment strategy to crd ([#1499](https://github.com/jaegertracing/jaeger-operator/pull/1499), [@ethernoy](https://github.com/ethernoy))
+* add cassandraCreateSchema affinity ([#1475](https://github.com/jaegertracing/jaeger-operator/pull/1475), [@chasekiefer](https://github.com/chasekiefer))
+* kafka 2.5.0 is not supported by RH AMQ operator 1.7.x ([#1476](https://github.com/jaegertracing/jaeger-operator/pull/1476), [@jkandasa](https://github.com/jkandasa))
```

Comment:
This entry needs to be rewritten to tell what the commit brought.

Reply:
May be omitting this entry? as I think is superseded by the upgrade commit to support 0.23.

Reply:
Sounds good.